Package.swift

```diff
@@ -101,13 +116,20 @@ let package = Package(
       ]
     ),
 
-    // "Support" targets: These contain C family code and are used exclusively
-    // by other targets above, not directly included in product libraries.
+    // "Support" targets: These targets are used exclusively by other targets
+    // above, not directly included in product libraries.
```

Comment:
> not directly included in product libraries

That's not accurate for `_TestDiscovery` right?

Reply:
Yeah, I can tweak the wording.